cmake/StaticBuild.cmake

```diff
@@ -319,7 +323,7 @@ build_external(unbound
   --enable-static --with-libunbound-only --with-pic
   --$<IF:$<BOOL:${WITH_LTO}>,enable,disable>-flto --with-ssl=${DEPS_DESTDIR}
   --with-libexpat=${DEPS_DESTDIR}
-  "CC=${deps_cc}" "CFLAGS=${deps_CFLAGS}"
+  "CC=${deps_cc}" "CFLAGS=${deps_CFLAGS}" "LDFLAGS=${deps_latomic}"
```

Comment:
better name: `deps_LDFLAGS`

Reply:
actually it was named like this before, but i remember seeing deps_ld somewhere

Reply:
one sec

Reply:
okay I think it should be fine to use deps_ld cause it's only used for android stuffs which is before where it sets the variable, i'll just re-use it https://github.com/oxen-io/lokinet/blob/bf0dc52df7c05d4a14b5d5140615c1d547430055/cmake/StaticBuild.cmake#L254

Reply:
no, that's the path to the linker itself.

Reply:
(we want to set additional linker flags)